src/veupath/redmine/client/rnaseq.py

```diff
@@ -228,9 +228,14 @@ def _parse_samples(self, sample_str: str) -> List[Dict]:
                             accessions_count[accession] += 1
                         else:
                             accessions_count[accession] = 1
-
-                    sample = {"name": self._normalize_name(sample_name), "accessions": accessions}
-                    samples.append(sample)
+                    norm_name = ""
+                    try:
+                        norm_name = self._normalize_name(sample_name)
+                        sample = {"name": norm_name, "accessions": accessions}
+                        samples.append(sample)
+                    except SamplesParsingException as err:
```

Comment:
```suggestion
                    except SamplesParsingException:
```
No need to create a variable if not used.